native/src/components/NavigationTile.tsx

```diff
@@ -33,74 +30,30 @@ const TileTitle = styled.Text`
   font-size: 11px;
   margin-bottom: 5px;
 `
+
 const StyledPressable = styled(Pressable)<{ width: number }>`
   padding: 10px 3px;
   width: ${props => props.width}px;
   align-items: center;
 `
-const NewsDot = styled.Text`
-  position: absolute;
-  top: ${-NEWS_DOT_RADIUS / 2};
-  end: ${-NEWS_DOT_RADIUS / 2};
-  text-align: center;
-  line-height: ${NEWS_DOT_RADIUS}px;
-  height: ${NEWS_DOT_RADIUS}px;
-  width: ${NEWS_DOT_RADIUS}px;
-  border-radius: ${NEWS_DOT_RADIUS / 2}px;
-  background-color: #ee5353;
-  color: #ffffff;
-  elevation: 5;
-  shadow-color: #000;
-  shadow-offset: 0px 2px;
-  shadow-opacity: 0.25;
-  shadow-radius: 3.84px;
-`
 
-const StyledIcon = styled(Icon)`
+const StyledIcon = styled(SimpleImage)`
   width: ${ICON_SIZE / Math.sqrt(2)}px;
   height: ${ICON_SIZE / Math.sqrt(2)}px;
 `
 
 type NavigationTileProps = {
   tile: TileModel
-  theme: ThemeType
   width: number
 }
 
-class NavigationTile extends React.Component<NavigationTileProps> {
-  getNewsDot(): ReactNode {
```

Comment:
Why delete all this NewsDot logic? I think it is a nice feature, maybe it would make more sense to fix it, than to completely delete it.

Reply:
The feature/implementation is from 2019 and was never used so far while we still had to maintain it. Unused code should be removed, not kept for a theoretical future use. If we actually want to have the feature, we can easily restore it from revision. Therefore I changed the commit history and made a specific commit to remove the news dot logic. Sorry for the force push!